buddypress/members/index-directory.php

```diff
@@ -775,30 +775,30 @@
                         <input type="hidden" value="<?php if(isset($_GET['tag']) && strlen($_GET['tag']) > 0): print trim($get_tag); endif; ?>" name="tag" id="user-tag" />
                         <input type="hidden" value="<?php if(isset($_GET['location']) && strlen($_GET['location']) > 0): print trim($location); endif; ?>" name="location" id="user-location" />
                         <input type="hidden" value="<?php if(isset($_GET['language']) && strlen($_GET['language']) > 0): print trim($get_language); endif; ?>" name="language" id="user-language" />
-                        <input type="text" name="u" id="members-search" class="members__search-input" placeholder="<?php print __("Search people", "community-portal"); ?>" value="<?php if($search_user): ?><?php print $search_user; ?><?php endif; ?>" />
+                        <input type="text" name="u" id="members-search" class="members__search-input" placeholder="<?php _e('Search people', 'community-portal'); ?>" value="<?php if($search_user): ?><?php print $search_user; ?><?php endif; ?>" />
                         </div>
-                        <input type="submit" class="members__search-cta" value="<?php print __("Search", "community-portal"); ?>" />
+                        <input type="submit" class="members__search-cta" value="<?php _e('Search', 'community-portal'); ?>" />
                     </form>
                 </div>
             </div>
         </div>
         <div class="members__container">
             <div class="members__filter-container members__filter-container--hidden">
-                <span><?php print __("Search criteria:", "community-portal"); ?></span>
+                <span><?php _e('Search criteria:', 'community-portal'); ?></span>
                 <div class="members__select-container">
-                    <label class="members__label"><?php print __("Location", "community-portal"); ?></label>
+                    <label class="members__label"><?php _e('Location', 'community-portal'); ?></label>
                     <select class="members__location-select">
-                        <option value=""><?php print __('Select', "community-portal"); ?></option>
+                        <option value=""><?php _e('Select', "community-portal"); ?></option>
```

Comment:
Community Portal needs to be changed to single-quotes

Reply:
Updated